packages/block-library/src/site-title/edit/index.js

```diff
@@ -15,6 +15,7 @@ import {
 	InspectorControls,
 	BlockControls,
 	useBlockProps,
+	__experimentalUseFontAppearanceStyles as useFontAppearanceStyles,
```

Comment:
Any reason for this to leave in the block editor if it's only used in the site-title block?

Maybe folks working on block tools can shed more light on this new toolbar. cc @aaronrobertshaw @andrewserong @jasmussen

Reply:
Ok, just saw that @aaronrobertshaw is already reviewing this PR :)

Reply:
👍 Re-reviewing this PR is on my radar for the next couple of days.

The logic behind the hook living in the block-editor package was that it is so closely tied to the typography block supports in that package. There's also the potential for it to be re-used across other blocks leveraging the font weight and style supports.

Reply:
Maybe we can start with it being "private" unless we're certain that it's an API we want to offer to third-party devs.